distributor-node/package.json

```diff
@@ -50,7 +50,7 @@
     "url-join": "^4.0.1",
     "winston": "^3.3.3",
```

Comment:
Should we maybe bump winston as well while we're at it?

Reply:
Ok. I tried to update the `winston` to the most recent version and it turned out that it contains breaking changes (so will have to update code at couple of places). So unless we absolutely want to update the dependency, I would say it's fine for now. WDYT?

Reply:
yes, let's ignore in that case